types/bwip-js/index.d.ts

```diff
@@ -34,7 +34,7 @@ declare namespace BwipJs {
         paddingheight?: number | undefined;
 
         monochrome?: boolean | undefined;
-        alttext?: boolean | undefined;
+        alttext?: string | undefined;
```

Comment:
This looks like a bool from [this code](https://github.com/metafloor/bwip-js/blob/9fc4a7dba6b190eabd6461cf3866a97e179afc23/src/exports.js#L236) but it's not. It's a string that is when present sets `includetext = true`

This PR fixes a bug.

Reply:
👍 

I have tested the fix too in my local env and I was able to change the text beneath a barcode. Got stuck trying to write a test in this project for it though 🤷

Reply:
I got tests working for this once many moons ago. I'll see if I can find some time today to script something.